akd/src/append_only_zks.rs

```diff
@@ -761,6 +760,16 @@ impl Azks {
         start_epoch: u64,
         end_epoch: u64,
     ) -> Result<AppendOnlyProof, AkdError> {
+        if self.get_latest_epoch() < end_epoch || end_epoch <= start_epoch {
+            return Err(AkdError::Directory(DirectoryError::InvalidEpoch(format!(
+                "Start epoch must be less than end epoch, and end epoch must be at most the latest epoch. \
+                Start epoch: {}, end epoch: {}, latest_epoch: {}.",
+                start_epoch,
+                end_epoch,
+                self.get_latest_epoch(),
+            ))));
+        }
```

Comment:
To avoid future clippy warnings, we should use inline these. For example:

```rust
        let latest_epoch = self.get_latest_epoch();
        if latest_epoch < end_epoch || end_epoch <= start_epoch {
            return Err(AkdError::Directory(DirectoryError::InvalidEpoch(format!(
                "Start epoch must be less than end epoch, and end epoch must be at most the latest epoch. \
                Start epoch: {start_epoch}, end epoch: {end_epoch}, latest_epoch: {latest_epoch}."
            ))));
        }
```

Reply:
Ah, cool -- thanks!